pkg/gpu/nvidia/mig/mig.go

```diff
@@ -50,6 +50,20 @@ var gpuPartitionSizeMaxCount = map[string]int{
 	"3g.71gb":  2,
 	"4g.71gb":  1,
 	"7g.141gb": 1,
+	//nvidia-b200, nvidia-gb200
+	"1g.23gb": 7,
```

Comment:
This MIG related changes seems come from a newly merged PR: https://github.com/GoogleCloudPlatform/container-engine-accelerators/pull/466/files?

Maybe you need to do a `merge master` to update your commit to be on top of the latest master branch?